app.py

```diff
@@ -15,22 +14,27 @@
            methods=['POST'],
            content_types=['application/octet-stream'])
 def exportBag():
+    """Expects a compressed (zipped) BagIt structure that
+    contains two TSV files named  "participant.tsv" and
+    "sample.tsv" in its "data" sub-directory. It creates
+    a new workspace in a FireCloud namespace, and uploads
+    the data into it.
+    """
     req_body = app.current_request.raw_body
     req_query_params = app.current_request.query_params
     req_headers = app.current_request.headers
     os.chdir('/tmp')
     with zipfile.ZipFile(io.BytesIO(req_body), 'r') as archive:
         archive.extractall()
-    tsv_fname = '/tmp/manifest_bag/data/manifest.tsv'
+    payload = []
+    payload.append('/tmp/manifest/data/participant.tsv')
+    payload.append('/tmp/manifest/data/sample.tsv')
     workspace = req_query_params['workspace']
     namespace = req_query_params['namespace']
     url = 'https://api.firecloud.org/api/workspaces'
-    # token = os.getenv("TOKEN", None)
-    token = req_headers['Authorization']
-    manifest_io = ManifestIO(tsv_fname, url, workspace, namespace, token)
+    auth = req_headers['Authorization']
+    manifest_io = ManifestIO(payload, url, workspace, namespace, auth)
     statusCode = manifest_io.import_tsv_to_fc()
```

Comment:
This does not return a status code, it returns a json object with two status codes. Hence why in next comment, until I realized this, I had proposed returning this value.

I think you should either return a tuple and then do something like
```
(participantStatusCode, sampleStatusCode) = manifest_io.import_tsv_to_fc()
```

Or have manifest_io.import_tsv_to_fc() return a single status code that is 200 if they all succeeded, and otherwise the status code of the first failure.